resources/views/page.blade.php

```diff
@@ -1,45 +1,145 @@
+{{-- ====================================================================== --}}
+{{-- PHP utility methods used on this blade template                        --}}
+{{-- ====================================================================== --}}
+
+@php
+
+/*
+ * Gets the body classes related to the configured layout options.
+ */
+function getBodyClasses()
+{
+    $body_classes = '';
+    $screen_sizes = ['xs', 'sm', 'md', 'lg', 'xl'];
+
+    // Add classes related to the "sidebar_mini" configuration.
+
+    if (config('adminlte.sidebar_mini', true) === true)
+        $body_classes .= 'sidebar-mini ';
+    elseif ($sidebar_mini_cfg == 'md')
```

Comment:
Where comes `$sidebar_mini_cfg` from?

Reply:
It comes from a previous implementation like this:
```
$sidebar_mini_cfg = config('adminlte.sidebar_mini', true);

if ($sidebar_mini_cfg === true)
    $body_classes .= 'sidebar-mini ';
elseif ($sidebar_mini_cfg == 'md')
```
Going to fix this... thanks!